docs/API.ReturnCodes.md

```diff
@@ -9,5 +9,6 @@ Return Code | Description
 204 | No more results
 404 | Infoton not found
 401 | Unauthorized user, or password does not match user name
+422 | The request has no effect on CM-Well data
```

Comment:
a more accurate phrasing is: "Request is semantically correct, but does not make sense in CM-Well"